src/logger/logger.cpp

```diff
@@ -0,0 +1,58 @@
+#include <logger/logger.hpp>
+
+namespace logger {
+// Constructors & Destructor
+Logger::Logger()
+    : stdout_loglevel_(loglevel_t::DEBUG), file_loglevel_(loglevel_t::NONE) {}
+
+Logger::Logger(const string& logfile, loglevel_t::e file_loglevel,
+               loglevel_t::e stdout_loglevel)
+    : stdout_loglevel_(stdout_loglevel), file_loglevel_(file_loglevel) {
+  ofs.open(logfile.c_str(), std::fstream::out | std::fstream::app);
```

Comment:
```suggestion
    : ofs(logfile.c_str(), std::fstream::out | std::fstream::app), stdout_loglevel_(stdout_loglevel), file_loglevel_(file_loglevel) {
```
생성자에서 이렇게 바로 되면 이렇게 하는게 더 좋을 것 같아요